bin/health.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Check that system daemons are running
+
+pid=$(pidof /sbin/rpcbind)
+if [ -z "$pid" ]
+then
+    echo "/sbin/rpcbind not running"
+    exit 1
+fi
+
+pid=$(pidof /sbin/rpc.statd)
+if [ -z "$pid" ]
+then
+    echo "/sbin/rpc.statd not running"
+    exit 1
+fi
+
+pid=$(pidof /usr/sbin/rpc.idmapd)
+if [ -z "$pid" ]
+then
+    echo "/usr/sbin/rpc.idmapd not running"
+    exit 1
+fi
+
+# Check that Node.js is running 
+# Future use: this script for Nagios might be useful:
+# https://exchange.nagios.org/directory/Plugins/Operating-Systems/Linux/check_nfs_health-2Esh/details
+
+pid=$(pidof /usr/local/bin/node)
```

Comment:
This simply checks that Node.js is still running, it doesn't check that the NFS service is working. For that, this Nagios NFS health-checking script might have some bits we can test: https://exchange.nagios.org/directory/Plugins/Operating-Systems/Linux/check_nfs_health-2Esh/details